### PR TITLE
Retail cleanup 2 #8364 ke

### PR DESCRIPTION
**Pull request by keichwa**
Formulas done

### CODE REVIEW

File: modules/retail/pages/retail-deploy-terminals.adoc [hunk 1]
```diff
@@ -18,26 +18,25 @@ For how to build OS images, see xref:administration:image-management.adoc[].
 
 [IMPORTANT]
 ====
-After you have registered new terminals, always check the {susemgr} {webui} to ensure your terminal has connected successfully to the branch server, and you have not connected directly to the {susemgr} server by mistake.
+After you have registered new terminals, always check the {susemgr} {webui} to ensure your terminals have connected successfully to the branch server, and not directly to the {susemgr} server by mistake.
```
Comment (Loquacity) [inline modules/retail/pages/retail-deploy-terminals.adoc]:
```suggestion
After you have registered new terminals, always check the {susemgr} {webui} to ensure your terminals have connected successfully to the branch server, and not directly to the {susemgr} Server by mistake.
```

File: modules/retail/pages/retail-formula-bind.adoc [hunk 2]
```diff
@@ -21,12 +21,13 @@ Zone 2 is the reverse zone of Zone 1.
 Its main purpose is to resolve IP addresses back to hostnames.
 Zone 2 is primarily needed for the correct determination of the FQDNs of the branch.
 
-
+// REMARK: this procedure is probably too long.  Where to split?
+// REMARK: are these list items (`*`) substeps?  Or what?  Confusing.
```
Comment (Loquacity) [inline modules/retail/pages/retail-formula-bind.adoc]: They are substeps: they're basically all the fields on a single dialog. I struggled to put this procedure together, as you can tell! I think I originally had the reverse settings as a separate procedure, but they've merged together again at some point.

File: modules/retail/pages/retail-formulas-intro.adoc [hunk 3]
```diff
@@ -3,20 +3,20 @@
...
 [IMPORTANT]
 ====
 If a formula uses the same name as an existing Salt state, the two names will collide, and could result in the formula being used instead of the state.
 Always check the names of states and formulas to avoid name collisions.
 ====
 
 Most formulas can be updated using the {susemgr} {webui}.
-Once you have made changes to your formula, ensure you apply the highstate to propagate your changes to the appropriate services.
+When you have made changes to your formula, ensure you apply the highstate.
+With the highstate you propagate your changes to the appropriate services.
```
Comment (Loquacity) [inline modules/retail/pages/retail-formulas-intro.adoc]:
```suggestion
The highstate propagates your changes to the appropriate services.
```

This sentence is possibly redundant, though?

Comment (keichwa) [reply modules/retail/pages/retail-formulas-intro.adoc]: Yes, but here in the intro it's ok to give more background info.

File: modules/retail/pages/retail-formula-bind.adoc (hunk 2 above)
Comment (keichwa) [reply modules/retail/pages/retail-formula-bind.adoc]: yes ;)  When I continued reviewing I saw that it is consistent and thus it makes sense.

**Comment by keichwa** [comment]
There was a confict.  I hope I got it right...

**Pull request merged**